specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-07-01-preview/elastic.json

```diff
@@ -0,0 +1,2763 @@
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeploymentInfoResponse"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeploymentInfo_List": {
+            "$ref": "./examples/DeploymentInfo_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/createOrUpdateExternalUser": {
+      "post": {
+        "tags": [
+          "DeploymentInfo"
+        ],
+        "summary": "Create User inside elastic deployment which are used by customers to perform operations on the elastic deployment",
+        "operationId": "ExternalUser_CreateOrUpdate",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Elastic External User Creation Parameters",
+            "schema": {
+              "$ref": "#/definitions/ExternalUserInfo"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ExternalUserCreationResponse"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ExternalUser_CreateOrUpdate": {
+            "$ref": "./examples/ExternalUserInfo.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/getBillingInfo": {
+      "post": {
+        "tags": [
+          "ConnectedResources"
+        ],
+        "summary": "Get marketplace and organization info mapped to the given monitor.",
+        "operationId": "BillingInfo_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
>/parameters/ApiVersionParameter [](http://example.com/codeflow?start=22&length=31)

Recommend getting this from v5 common-types. Comment applies throughout this file.

Reply:
We are already using this from before. And there is no change here from previous version. Therefore, will not touch it.

Reply:
I would still insist to start using common-type and delete local definition of these parameters since it is a new api-version if there is no other reason than using these in earlier versions. At some point everyone need to move to common definition for the goodness it brings.

Reply:
Updated. Now fetching subscription, resource group and api version from v5 common types